rice-middleware/impl/src/main/java/edu/arizona/rice/kim/service/impl/UaUiDocumentServiceImpl.java

```diff
@@ -3,7 +3,7 @@
  * Copyright 2009 State of Arizona Board of Regents
  *
  */
-package edu.arizona.kim.service.impl;
+package edu.arizona.rice.kim.service;
```

Comment:
Ok. Let's leave it for now... Just make sure the package name matches the folder where the file is in to avoid compilation errors:
package edu.arizona.rice.kim.service.impl;